pandas/core/frame.py

```diff
@@ -5005,39 +5005,41 @@ def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
 
     def applymap(self, func):
         """
-        Apply a function to a DataFrame that is intended to operate
-        elementwise, i.e. like doing map(func, series) for each series in the
-        DataFrame
+        Apply a function to a Dataframe elementwise.
+
+        This method applies a function to a DataFrame that is intended to
+        operate elementwise, i.e. like doing map(func, series) for each series
+        in the DataFrame.
 
         Parameters
         ----------
         func : function
-            Python function, returns a single value from a single value
-
-        Examples
-        --------
-
-        >>> df = pd.DataFrame(np.random.randn(3, 3))
-        >>> df
-            0         1          2
-        0  -0.029638  1.081563   1.280300
-        1   0.647747  0.831136  -1.549481
-        2   0.513416 -0.884417   0.195343
-        >>> df = df.applymap(lambda x: '%.2f' % x)
-        >>> df
-            0         1          2
-        0  -0.03      1.08       1.28
-        1   0.65      0.83      -1.55
-        2   0.51     -0.88       0.20
+            Python function, returns a single value from a single value.
 
         Returns
         -------
-        applied : DataFrame
+        pandas.DataFrame
+            A transformed DataFrame.
 
         See also
         --------
-        DataFrame.apply : For operations on rows/columns
+        DataFrame.apply : Apply a function along input axis of DataFrame
+
+        Examples
+        --------
 
+        >>> df = pd.DataFrame(np.random.RandomState(0).randn(3, 3))
```

Comment:
In the [Conventions for the examples](https://python-sprints.github.io/pandas/guide/pandas_docstring.html#conventions-for-the-examples) mentions:
> avoid using data without interpretation, like a matrix of random numbers with columns A, B, C, D… And instead use a meaningful example, which makes it easier to understand the concept

In fact, this one looks like the **Bad** example :-P

I'd suggest improving this example.

Reply:
thanks for the comment! We upgraded the original example to a deterministic one (adding the hardcoded seed in RandomState),  .. any idea to improve the example?

Reply:
thanks for the comment, actually we updated previous random example to a deterministic one by hardcoding the random seed through RandomState(0). Since we are illustrating an elementwise mapping it might be easier to use a matrix [[1,2],[3,4]] and square it.